.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contribution guidelines
+
+We're happy to accept 3rd-party contributions. Please make sure you read this document before you do any work though,
+as we have some expectations related to the content and quality of change sets.
+
```

Comment:
We should probably have a very short description about what this application is for

Reply:
that's what readme is for?

Reply:
fair enough